### PR TITLE
Update Bendigo Bank

### PR DESCRIPTION
**Pull request by james-maclean**
Bendigo Bank now supports MFA using their iOS and Android application.

### CODE REVIEW

File: entries/b/bendigobank.com.au.json [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
   "Bendigo Bank": {
     "domain": "bendigobank.com.au",
-    "contact": {
-      "facebook": "BendigoBank",
-      "twitter": "bendigobank",
-      "form": "https://www.bendigoadelaide.com.au/contact-us/customer-feedback/"
-    },
+    "tfa": [
+      "custom-software"
+    ],
```
Comment (hkamran80) [inline entries/b/bendigobank.com.au.json:8]:
```suggestion
    ],
    "custom-software": [
      "Bendigo Bank app"
    ],
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]
If you enable MFA through their app, does it stop Symantec VIP Access and the physical tokens from working? I've seen some of their videos say they support SMS and email tokens. Is that true?

**Comment by james-maclean** [comment]
Yes, if you enable MFA in the app (soon to be forced) it disables the physical token and Symantec VIP. 

These options were never forced on login, the new MFA is. I can't find anything to imply email or sms are supported methods.

**Comment by hkamran80** [comment]
> These options were never forced on login, the new MFA is. I can't find anything to imply email or sms are supported methods.

It says in each of the videos next to the documentation link you added. Here's a [direct link](https://youtu.be/iRqHTWDLavs?t=78).

File: entries/b/bendigobank.com.au.json [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
+    "custom-software": [
+      "Bendigo Bank app"
+    ],
+    "documentation": "https://www.bendigobank.com.au/security/e-banking-upgrade/#tab-241991",
```
Comment (hkamran80) [inline entries/b/bendigobank.com.au.json]:
```suggestion
    "documentation": "https://www.bendigobank.com.au/support/e-banking/multi-factor-authentication",
```

I think this is a better page to use for documentation

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/b/bendigobank.com.au.json (hunk 2 above)
Comment (james-maclean) [reply entries/b/bendigobank.com.au.json]: It didn't feel focused enough for me but I'm happy to change it.

**Comment by james-maclean** [comment]
Okay, so you are correct. It does not offer the option of App, and SMS/Email it's App OR SMS/Email.

File: entries/b/bendigobank.com.au.json [hunk 3]
```diff
@@ -1,16 +1,18 @@
 {
   "Bendigo Bank": {
     "domain": "bendigobank.com.au",
-    "contact": {
-      "facebook": "BendigoBank",
-      "twitter": "bendigobank",
-      "form": "https://www.bendigoadelaide.com.au/contact-us/customer-feedback/"
-    },
+    "tfa": [
+      "custom-software"
```
Comment (hkamran80) [inline entries/b/bendigobank.com.au.json:7]:
```suggestion
      "sms",
      "email",
      "custom-software"
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/b/bendigobank.com.au.json [hunk 4]
```diff
@@ -1,16 +1,18 @@
...
+    ],
+    "custom-software": [
+      "Bendigo Bank app"
+    ],
+    "documentation": "https://www.bendigobank.com.au/support/e-banking/multi-factor-authentication",
```
Comment (hkamran80) [inline entries/b/bendigobank.com.au.json:12]:
```suggestion
    "documentation": "https://www.bendigobank.com.au/support/e-banking/multi-factor-authentication",
    "notes": "If the app hasn't been setup, only SMS and email tokens will work. If the app has been setup, only the app will work.",
```

Is this is an accurate representation of their MFA support?

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/b/bendigobank.com.au.json (hunk 4 above)
Comment (james-maclean) [reply entries/b/bendigobank.com.au.json:12]: Accurate.

**Review by Carlgo11 — APPROVED** [review]

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**